### DISCUSSION TITLE
Welcome to step-series-forum Discussions!

### DISCUSSION THREAD

**Discussion by kanta** [ponoor/step-series-support#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kanta** [comment]
このリポジトリのDiscussionsをフォーラムとして使っていこうと思います。何か不都合が生じるまでは、日本語も英語も混ぜた状態でいいかなと思っています。

I'm going to use this repository's Discussions as a forum. You can use English or Japanese in any discussions so far.

**Comment by florianillumination** [comment]
Hello,
We are building a motorised slider for a camera and we have a small issue.
We feel step during accelerating and decelerating. It is ok during normal speed. We are using our step motor very slow and we need a movement as smooth as possible and as quiet as possible.
You can see the issue in the video attached.

Thank you in advance for your help.

Florian

We have a STEP400.

The setup is:
/setKval 1 15 30 30 30
/setSpeedProfile 1 5 5 30  , then 1 15 15 50 , then 1 20 20 100
/run 1 100

INT_SPEED 1032
ST_SLP 25
FN_SLP_ACC 41
FN_SLP_DEC 41

Power Supply: 24V - 13A

Motor information:
https://www.omc-stepperonline.com/nema-23-bipolar-1-8deg-1-16nm-164-3oz-in-1-5a-5-4v-57x57x56mm-4-wires.html 

https://user-images.githubusercontent.com/98390269/150960886-155013ac-9707-4029-8be2-8387096eb0fa.mp4

**Reply by kanta** [reply]
Hello @florianillumination , this thread is a part of welcome message so I quoted your post to here #7 . Let's continue there!
